modules/runner/src/integrationTest/java/org/apache/ignite/internal/ssl/ItSslTest.java

```diff
@@ -156,11 +161,11 @@ class ClusterWithSsl {
                 + "      enabled: true,\n"
                 + "      trustStore: {\n"
                 + "        password: \"" + password + "\","
-                + "        path: \"" + trustStorePath + "\""
+                + "        path: \"" + trustStorePath.replace("\\", "\\\\") + "\""
```

Comment:
Could you please move `.replace("\\", "\\\\")` to the self-describing method like `escape(trustStorePath)` or something like this?

Reply:
Moved to `escapeForWindows`.